### PR TITLE
:tada: first implementation

### PR DESCRIPTION
**Pull request by fabienjuif**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+# trampss-mst-onaction
+
+Listen to mobx-state-tree action and react to them !
```
Comment (delphinemillet) [inline README.md]: action`s`

File: README.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+Example: we want a login action to trigger a router action.
+We can do this this way :
+```es6
+import onAction from 'trampss-mst-onaction'
+import Store from './your-store-model'
+
+// instanciate the store
+const store = Store.create({})
+
+// the actions to trigger
```
Comment (delphinemillet) [inline README.md:44]: action (without s)

**Review by delphinemillet — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (fabienjuif) [reply README.md:44]: This is where you set all actions you want to listen to trigger new ones.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+## Installation
+ - `yarn add trampss-mst-onaction`
+ - `npm i trampss-mst-onaction`
+
+## API
+TODO
+
+Example: we want a login action to trigger a router action.
+We can do this this way :
```
Comment (delphinemillet) [inline README.md]: do it

File: README.md (hunk 2 above)
Comment (delphinemillet) [reply README.md:44]: ok, I wasn't sure

**Review by delphinemillet — APPROVED** [review]

**Pull request merged**